src/coreclr/jit/targetloongarch64.h

```diff
@@ -110,7 +110,6 @@
   // Temporary registers used for the GS cookie check.
   #define REG_GSCOOKIE_TMP_0       REG_T0
   #define REG_GSCOOKIE_TMP_1       REG_T1
-  #define RBM_GSCOOKIE_TMP         (RBM_T0|RBM_T1)
```

Comment:
why is this deleted?

Reply:
Because this is only used in the above change in `lsraloongarch64.cpp`, now we use `~(genSingleTypeRegMask(REG_GSCOOKIE_TMP_0) | genSingleTypeRegMask(REG_GSCOOKIE_TMP_1))` instead, so we can delete this now.